examples/src/main/res/values/strings.xml

```diff
@@ -16,11 +16,14 @@
     <string name="title_onboard_router_java" translatable="false">Onboard Router Java</string>
     <string name="description_onboard_router_java" translatable="false">Shows routes using onboard routing api in Java code.</string>
 
-    <string name="title_trip_service_kotlin">Trip Service Kotlin</string>
-    <string name="description_trip_service_kotlin">Shows how to use the Service to push notifications in Kotlin code.</string>
+    <string name="title_trip_service_kotlin" translatable="false">Trip Service Kotlin</string>
```

Comment:
Why `translatable="false"`? We currently have the test app project set up in Transifex and translators are contributing to it, we should do the same for `examples` module.

Reply:
Why do we need translations for an example/debug project? It looks like we could direct that effort somewhere else.